src/UserSettingsPage.cpp

```diff
@@ -464,6 +465,16 @@ UserSettings::setHomeserver(QString homeserver)
         save();
```

Comment:
Recommend adding checks to the save and load sections to check if the jdenticon is available.  If it is not, load/save should set it to 'false' and log a warning to the user

Reply:
See above comment.

Reply:
What is this 'above' comment... not seeing it

Reply:
https://github.com/Nheko-Reborn/nheko/pull/416#discussion_r564134077